RELEASE.md

```diff
@@ -1,3 +1,76 @@
+# Release 1.11.0
+
+## Major Features and Improvements
+
+* Prebuilt binaries are now (as of TensorFlow 1.11) built against cuDNN 7.2. See updated install guides: [Installing TensorFlow on Ubuntu](https://www.tensorflow.org/install/install_linux#tensorflow_gpu_support)
+* Cloud TPU:
```

Comment:
Change "Cloud TPU" here and in next line to "Google Cloud TPU".

Reply:
Done, thanks!